pages/api/scopes/index.ts

```diff
@@ -36,13 +39,27 @@ export default async function handler(
           return;
         }
 
+        const user_uuid = user.uuid;
+
         await prisma.scopes.create({
-          data: {
-            user_uuid: user?.uuid,
-            scope,
-          },
+          data: { user_uuid, scope },
         });
 
+        if (scope === BANNED) {
+          await prisma.users_stripe_products.deleteMany({
```

Comment:
* Just to speed this up, let's put it in a `Promise.all` so it runs in parallel
* Also will want to update user prop `club_uuid` to null